@here/harp-mapview/test/VisibleTileSetTest.ts

```diff
@@ -51,11 +52,10 @@ class Fixture {
             MapViewDefaults.extendedFrustumCulling,
             params.tileWrappingEnabled
         );
-        this.vts = new VisibleTileSet(
-            this.frustumIntersection,
-            this.tileGeometryManager,
-            MapViewDefaults
-        );
+        this.vts = new VisibleTileSet(this.frustumIntersection, this.tileGeometryManager, {
+            ...MapViewDefaults,
+            clipPlanesEvaluator: new TiltViewClipPlanesEvaluator()
```

Comment:
Somehow it shows the need for default factory method  ... but as you wish....

Reply:
ok, done